src/utils/misc.ts

```diff
@@ -20,6 +20,15 @@ export function mod(a: number, n: number) {
   return (a % n + n) % n;
 }
 
+/**
+ * Returns the sum of the numbers in array.
+ *
+ * @param array The array of number to iterate over
+ */
+export function sum(array: number[]): number {
```

Comment:
test for these helper methods?

Reply:
This helper method will be removed and replaced with `mathjs.sum`